### DISCUSSION TITLE
Learning next.js 13 and getting an error prompt on terminal

### DISCUSSION THREAD

**Discussion by thatshubham** [vercel/next.js#46530] [Help]
Hi guys, I'm using the new `app` directory for a self-learning project and keep getting this error message on the terminal, however the application compiles fine : 

`Do not use <head> element. Use <Head /> from next/head instead. See: https://nextjs.org/docs/messages/no-head-element`

The fast-refresh also seldom works. Could there be anything wrong in my implementation? 

---

For context, here's my `layout.js`
```

export default function RootLayout({ children }) {
  return (
    <html lang="en">
      {/*
        <head /> will contain the components returned by the nearest parent
        head.js. Find out more at https://beta.nextjs.org/docs/api-reference/file-conventions/head
      */}
      <head />

      <body class="flex items-center">
      <Navbar />
      {children}
      <Footer />
      </body>
    </html>
  )
}
```

Thank you,
Shubham

**Comment by kfirfitousi** [comment]
Are you using `<head>` somewhere inside `<Navbar>`, `<Footer>`, or your `page.js`?
If so, you should use `<Head>` instead:
```jsx
import Head from 'next/head';

export function Navbar() {
  return (
    <>
      <Head>
        <title>Page Title</title>
      </Head>
            
      <nav>
        {/* ... */}
      </nav>
    </>
  ) 
}
```

**Reply by thatshubham** [reply]
@kfirfitousi Hi, thanks for following-up. Nope, I'm not using `<head>` anywhere outside `layout.js` in my application.

**Reply by kfirfitousi** [reply]
Which version of `next` are you using? Can you reproduce this in a fresh `create-next-app@latest` project?

**Reply by thatshubham** [reply]
I'm using version **13.1.6**. Here are my dependencies : 
```
      "dependencies": {
        "@next/font": "13.1.6",
        "eslint": "8.34.0",
        "eslint-config-next": "13.1.6",
        "next": "13.1.6",
        "react": "18.2.0",
        "react-dom": "18.2.0"
      },
      "devDependencies": {
        "autoprefixer": "^10.4.13",
        "postcss": "^8.4.21",
        "tailwindcss": "^3.2.7"
      }
```

Hmm, thanks for the suggestion. I'll try reproducing this in a new project and report back in some time.

**Reply by kfirfitousi** [reply]
I'm not sure if this applies to your case, but during my tests I've encountered an issue with `eslint-plugin-next`.
Opened an issue #46559 and a PR to fix it #46560
